.github/workflows/publish-package.yml

```diff
@@ -0,0 +1,29 @@
+name: publish-package
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: google-github-actions/release-please-action@v3
+        with:
+          release-type: node
+          package-name: "@ubiquitydao/rpc-handler"
+          default-branch: main
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
```

Comment:
`checkout` and `setup-node` can be using latest version (v4)

Reply:
I recommend you suggest and commit like I do straight from the GitHub UI. Try it out!

Reply:
leaving mine up until the ubq one is published so I can move forward with other PRs in the meantime